### ISSUE TITLE
Update providers.json

### ISSUE THREAD

**Issue by rdehouss** [jasontbradshaw/gandi-dyndns#7]
Hello,

Great tool!

Some IP providers are not working anymore and could be replaced by others:

```
2016-10-14 11:20:55,780 [WARNING] Error getting external IP address from http://curlmyip.com: <urlopen error [Errno 111] Connection refused>
2016-10-14 11:20:55,814 [WARNING] Error getting external IP address from http://ifconfig.sh: <urlopen error [Errno -2] Name or service not known>
2016-10-14 11:20:55,868 [WARNING] Error getting external IP address from http://ipchicken.com: HTTP Error 403: Forbidden
2016-10-14 11:20:56,089 [WARNING] Error getting external IP address from http://ipconfig.io: <urlopen error [Errno -2] Name or service not known>
2016-10-14 11:20:57,101 [WARNING] Error getting external IP address from http://www.whatsmyip.us: <urlopen error [Errno 113] No route to host>
```

New ones:
http://whatismyip.akamai.com/ (very fast)
https://ifconfig.co/

Cheers,

Raphaël Dehousse

**Comment by jasontbradshaw** [comment]
Good to know!

If you'd like to open a PR to update the list, I'd be happy to merge it.

**Comment by jasontbradshaw** [comment]
Merged #9, thanks @rdehouss!
